### DISCUSSION TITLE
Is Mongoose v6.x compatible with MongoDB v6?

### DISCUSSION THREAD

**Discussion by DavideViolante** [Automattic/mongoose#13078] [Q&A] [answered]
I received an email from MongoDB Atlas Cloud that they are upgrading to MongoDB v6. I wanted to be sure that Mongoose is compatible.

Thanks.

**Comment by vkarpov15** [comment, marked as answer]
Mongoose ^6.5.0 is compatible with MongoDB 6. Take a look at [Mongoose's version compatibility doc](https://mongoosejs.com/docs/compatibility.html) for up-to-date info on what versions of Mongoose support what versions of the MongoDB server.
